src/test/kotlin/uk/gov/justice/digital/hmpps/breachnoticeapi/integration/BreachNoticeCrudTests.kt

```diff
@@ -0,0 +1,230 @@
+package uk.gov.justice.digital.hmpps.breachnoticeapi.integration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.breachnoticeapi.entity.BreachNoticeEntity
+import uk.gov.justice.digital.hmpps.breachnoticeapi.model.Address
+import uk.gov.justice.digital.hmpps.breachnoticeapi.model.BreachNotice
+import uk.gov.justice.digital.hmpps.breachnoticeapi.repository.BreachNoticeRepository
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class BreachNoticeCrudTests : IntegrationTestBase() {
+
+  @Autowired
+  private lateinit var breachNoticeRepository: BreachNoticeRepository
+
+  @Test
+  fun `should create a breach notice`() {
+    webTestClient.post()
+      .uri("/breach-notice")
+      .headers(setAuthorisation(roles = listOf("ROLE_TEMPLATE_KOTLIN__UI")))
+      .bodyValue(
+        BreachNotice(
+          crn = "X00000B",
+        ),
+      )
+      .exchange()
+      .expectStatus()
+      .isCreated
+
+    var breachNotice: MutableList<BreachNoticeEntity> = breachNoticeRepository.findByCrn("X00000B")
+    assertThat(breachNotice.first().crn).isEqualTo("X00000B")
+    assertThat(breachNotice.first().id).isNotNull()
```

Comment:
Minor thing - you could use [.single()](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.collections/single.html) if you want to be certain only a single result is returned.

Also make sure to use `val`s where possible.

```suggestion
    val breachNotice = breachNoticeRepository.findByCrn("X00000B").single()
    assertThat(breachNotice.crn).isEqualTo("X00000B")
    assertThat(breachNotice.id).isNotNull()
```